src/Components/CreateImageWizardV2/validators.ts

```diff
@@ -36,3 +38,24 @@ export const isBlueprintNameValid = (blueprintName: string) =>
 export const isBlueprintDescriptionValid = (blueprintDescription: string) => {
   return blueprintDescription.length <= 250;
 };
+export const isFileSystemConfigValid = (partition: Partition[]) => {
```

Comment:
Let's refactor this a bit so it is consistent, and change `partition` to `partitions`.